src/main/resources/init_db.sql

```diff
@@ -0,0 +1,11 @@
+create schema taxi_service collate utf8_general_ci;
+
+create table manufacturers
+(
+    id         bigint auto_increment
+        primary key,
+    name       varchar(255)      not null,
+    country    varchar(255)      not null,
+    is_deleted tinyint default 0 not null
+);
+
```

Comment:
Does this works for you?

```suggestion
CREATE DATABASE `taxi_service` DEFAULT CHARACTER SET utf8 DEFAULT ENCRYPTION='N';

CREATE TABLE `manufacturers` (
  `id` bigint NOT NULL AUTO_INCREMENT,
  `name` varchar(255) DEFAULT NULL,
  `country` varchar(255) DEFAULT NULL,
  `is_deleted` tinyint NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;

```